config/routes.rb

```diff
@@ -61,6 +61,8 @@
 
   get 'cms/youtube_embed/:id', to: 'cms/youtube_embed#show'
 
+  get '/user/school(/*path)', to: 'redirects#school_page_redirect', as: :school_page_redirect, constraints: { path: /.*/ }
```

Comment:
wonder about
/my/path
/my/school/path
/redirect/path
/shortened/path
but as you say can change later